bedrock/mozorg/templates/mozorg/MPL/1.1/index.html

```diff
@@ -0,0 +1,389 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html lang="en">
+<head>
+ <title>Mozilla Public License version 1.1</title>
+ <style type="text/css">
+  .very-strong{
+   text-transform:uppercase;
+  }
+  dt{
+   font-weight:bold;
+  }
+  dd p{
+   margin:0;
+  }
+ </style>
+</head>
+<body>
+ <p><small>(<a href="/MPL/1.1/index.txt">Plain text version</a>)</small></p>
```

Comment:
I notice this was a relative URL in the SVN version (`href="index.txt"`). Can it stay that way?